src/assets/style.css

```diff
@@ -152,13 +167,39 @@
 }
 
 .wizard--container {
-  width: 30%;
+  width: 40%;
   margin: auto;
   text-align: center;
   display: flex;
   flex-direction: row;
   flex-wrap: wrap;
   justify-content: space-between;
+  padding: 10px;
+}
+
+.wizard--container-board {
+  width: 100%;
+  height: 500px;
+  overflow: scroll;
+  scroll-behavior: smooth;
+}
+
+.wizard--container-board::-webkit-scrollbar {
+    -webkit-appearance: none;
+}
+
+.wizard--container-board::-webkit-scrollbar:vertical {
+    width: 11px;
+}
+
+.wizard--container-board::-webkit-scrollbar:horizontal {
+    height: 11px;
+}
+
+.wizard--container-board::-webkit-scrollbar-thumb {
+    border-radius: 8px;
```

Comment:
Expected indentation of 2 spaces                                                indentation